README.md

```diff
@@ -0,0 +1,110 @@
+# Scalar Admin k8s
+
+## Usage of the CLI tool
+
+```console
+Usage: scalar-admin-k8s-cli [-h] [-d=<pauseDuration>] [-n=<namespace>]
+                            -r=<helmReleaseName> [-w=<maxPauseWaitTime>]
+                            [-z=<zoneId>]
+Scalar Admin pause tool for the Kubernetes environment
+  -d, --pause-duration=<pauseDuration>
+                             The duration of the pause period by millisecond.
+                               5000 (5 seconds) by default.
+  -h, --help                 Display the help message.
+  -n, --namespace=<namespace>
+                             Namespace that Scalar products you want to pause
+                               are deployed. `default` by default.
+  -r, --release-name=<helmReleaseName>
+                             Helm's release name that you specify when you run
+                               the `helm install <RELEASE_NAME>` command. You
+                               can see the <RELEASE_NAME> by using the `helm
+                               list` command.
+  -w, --max-pause-wait-time=<maxPauseWaitTime>
+                             The max wait time (in milliseconds) until Scalar
+                               products drain outstanding requests before they
+                               pause. If omitting this option, the max wait
+                               time will be the default value defined in the
+                               products. Most Scalar products have the default
+                               value of 30 seconds.
+  -z, --time-zone=<zoneId>   Specify a time zone ID, e.g., Asia/Tokyo, to
+                               output successful paused period. Note the time
+                               zone ID is case sensitive. Etc/UTC by default.
+```
+
+## Run the CLI tool on a Kubernetes environment
+
+The `scalar-admin-k8s` CLI tool executes Kubernetes APIs in its internal processes. To run those Kubernetes APIs, you must run the `scalar-admin-k8s` CLI tool as a pod on the Kubernetes environment and you must:
```

Comment:
Depending on what we call the product (as mentioned in my previous comment), the product mentions in this paragraph might need to be updated.

```suggestion
The `scalar-admin-k8s` CLI tool executes Kubernetes APIs in its internal processes. To run those Kubernetes APIs, you must run the `scalar-admin-k8s` CLI tool as a pod on the Kubernetes environment and then do the following:
```

Reply:
@josh-wong  
Thank you for your suggestion! 

In practice, the content of this section is a `steps` to deploy Scalar Admin for Kubernetes CLI on the Kubernetes environment.

So, how about the following sentence?
```suggestion
The `scalar-admin-for-kubernetes` CLI tool executes Kubernetes APIs in its internal processes. To run those Kubernetes APIs, you must run the `scalar-admin-for-kubernetes` CLI tool as a pod on the Kubernetes environment by following to below steps.
```

Reply:
@kota2and3kan Thank you for clarifying that! I assumed the steps were separate from the `you must run` part since we use `and` to introduce the steps.

In this case, let's use the following:

```suggestion
The `scalar-admin-for-kubernetes` CLI tool executes Kubernetes APIs in its internal processes. To run those Kubernetes APIs, you must run the `scalar-admin-for-kubernetes` CLI tool as a pod on the Kubernetes environment by following the steps below:
```

Reply:
@josh-wong  
Thank you for your suggestion!
I applied it in 8d5d2a4e3a8686ecba1c1607e98224988f547ec4 .